### DISCUSSION TITLE
How do I tweet 1 out of 3 tweets at a 15 sec interval

### DISCUSSION THREAD

**Discussion by f3bw** [tweepy/tweepy#1377] [Questions] [answered]
Hello, I know how to create a 15 second interval, but I can't figure out how to send only 1 of my 3 api.update_status without sending the other ones. Can anybody help me out?

**Comment by ReubenMathew** [comment]
Make a queue, add each of the 3 tweets to the queue. When the queue reaches size of 3 send off the first element and delete the queue so that it is refreshed.

**Comment by Harmon758** [comment, marked as answer]
I'm not sure what exactly the issue is.
You can simply sleep the 15 seconds between each call to `API.update_status`.
You'll need to provide what exactly you're trying if you need further help.
